src/Model/Repository/BaseRepository.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+
+namespace Friendica\Model\Repository;
+
+use Friendica\Database\Database;
+use Friendica\Model\Entity\BaseEntity;
+use Friendica\Model\Entity\IStorable;
+
+abstract class BaseRepository
```

Comment:
This is the base of each repository. Every repository, which interacts with a table in Friendica has to extend this class